test/workbox-routing/node/Route.mjs

```diff
@@ -91,43 +91,43 @@ describe(`workbox-routing: Route`, function() {
   });
 
   it(`should use the method provided when called with a valid method in dev`, function() {
-    if (process.env.NODE_ENV == 'production') return this.skip();
+    if (process.env.NODE_ENV === 'production') return this.skip();
 
     const route = new Route(match, handler, method);
-    expect(route._method).to.equal(method);
+    expect(route.method).to.equal(method);
```

Comment:
If you use `.method` the minifier won't be able to manipulate the value. Do we need it to be `.method` and public facing?

Reply:
I can switch.